src/Uno.UI.Runtime.Skia.Wpf/Extensions/WpfDragDropExtension.cs

```diff
@@ -244,7 +244,7 @@ private static async Task<DataObject> ToDataObject(DataPackageView src, Cancella
 			return dst;
 		}
 
-		private class DragEventSource : IDragEventSource
+		private readonly struct DragEventSource : IDragEventSource
```

Comment:
Note: there is no perf benefit here as it will be passed to `CoreDragInfo` which takes IDragEventSource. So, boxing is going to happen. Probably better to not change this.

Reply:
This is not a perf change. This is a correctness change. The only way to make a class read-only is to make it a struct even if you know it will be boxed.

Reply:
The only state I'm seeing is _wpfArgs which is readonly already. Not sure I'm following

Reply:
The only state you're seeing _for now_ is _wpfArgs. Making the entire thing readonly makes sure that whoever decides to add new state will need to think twice before removing `readonly`.